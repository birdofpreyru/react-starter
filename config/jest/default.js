const config = require('topcoder-react-utils/config/jest/default');

module.exports = config;
