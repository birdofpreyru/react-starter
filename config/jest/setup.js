import '@dr.pogodin/react-utils/config/jest/setup';
