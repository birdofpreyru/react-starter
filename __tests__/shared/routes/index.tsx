/** @jest-environment jsdom */

import { snapshot } from '@dr.pogodin/react-utils/jest';

import Routes from 'routes';

test('Matches shallow snapshot', () => {
  snapshot(<Routes />);
});
