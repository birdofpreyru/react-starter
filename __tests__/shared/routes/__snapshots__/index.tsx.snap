// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<span
  class="_7zdld4"
>
  <span
    class="dBrB4g"
  />
  <span
    class="dBrB4g"
  />
  <span
    class="dBrB4g"
  />
</span>
`;
